proposals/nnnn-structured-concurrency.md

```diff
@@ -209,6 +255,124 @@ If the scope of an `async let` exits with a thrown error, the child task corresp
 
 > **Rationale**: The requirement to await a variable from each `async let` along all (non-throwing) paths ensures that child tasks aren't being created and implicitly cancelled during the normal course of execution. Such code is likely to be needlessly inefficient and should probably be restructured to avoid creating child tasks that are unnecessary.
  
+ 
+### Detached Tasks
+
+Detached tasks are one of the two "escape hatch" APIs offered in this proposal (the other being the `UnsafeContinuation` APIs discussed in the next section), for when structured concurrency rules are too rigid for a specific asynchronous operations.
+
+
+Looking at the previously mentioned example of making dinner in a detached task, but fillin in the missing types and details:
+
+
+```swift
+let dinnerHandle: Task.Handle<Dinner, Never> = Task.runDetached {
+  await makeDinner()
+}
+
+// optionally, someone, somewhere may cancel the task:
+// dinnerHandle.cancel()
+
+let dinner = await try dinnerHandle.get()
+```
+
+The `Task.Handle` returned from the `runDetached` function serves as a reference to an in-flight `Task`, allowing either awaiting or cancelling the task.
+
+It is important that the get() can be not only of the expected `Task.Handle.Failure` type, but also the `CancellationError`, so awaiting on a `handle.get()` is *always* throwing, even if the wrapped operation was not throwing itself.
```

Comment:
We discussed this @DougGregor but I'm actually not sure about it anymore... in fact, I think we got it wrong and a revival of the non-throwing `get` for `Handle where Failure == Never` is needed.

Because:

```
handle.cancel() // task/thread A:
// cancel ONLY means that the cancelled flag on the task is set to `true`

{ // running the handle's task on B
   // it does not check for cancellation / thus it will never throw or "be cancelled"
   do whatever()
}

await handle.get() // task/thread C:  MAY throw but may also not throw, since cancellation is co-operative
```